cmd/neofs-node/reputation.go

```diff
@@ -39,6 +47,133 @@ type localTrustIterator struct {
 	epochStorage *truststorage.EpochTrustValueStorage
 }
 
+type managerBuilder struct {
+	log   *logger.Logger
+	nmSrc netmapcore.Source
+}
+
+type remoteLocalTrustProvider struct {
+	loadAddrSrc network.LocalAddressSource
+
+	clientCache interface {
+		Get(string) (apiClient.Client, error)
+	}
+}
+
+type nopReputationWriter struct{}
+
+func (nopReputationWriter) Write(reputation.Trust) error {
+	return nil
+}
+
+func (nopReputationWriter) Close() error {
+	return nil
+}
+
+type remoteLocalTrustWriter struct {
+	ctx    trustcontroller.Context
+	client apiClient.Client
+
+	buf []*reputationapi.Trust
+}
+
+func (rtp *remoteLocalTrustWriter) Write(t reputation.Trust) error {
+	val := t.Value().Float64()
+	peer := t.Peer().Bytes()
+
+	apiTrust := reputationapi.NewTrust()
+
+	apiTrust.SetValue(val)
+	apiTrust.SetPeer(reputationapi.PeerIDFromV2(peer))
```

Comment:
Pls update `neofs-api-go` lib, there is a new API in `reputation` pkg there.

Reply:
After https://github.com/nspcc-dev/neofs-node/pull/452 you can just rebase to get latest api-go version.

Reply:
rebased and used new api version